From 47ee563f88d541dc20b83ab8edf2b2430c17dd58 Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.com>
Date: Mon, 17 Feb 2025 11:38:32 +0000
Subject: [PATCH] Mixins changes to enable wlan on IA

This patch will enable wlan on IA on A16

Tests Done: wlan is working fine

Tracked-On: NA
Signed-off-by: JeevakaPrabu <jeevaka.badrappan@intel.com>
---
 groups/wlan/iwlwifi/BoardConfig.mk | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/groups/wlan/iwlwifi/BoardConfig.mk b/groups/wlan/iwlwifi/BoardConfig.mk
index 2293331..6383277 100644
--- a/groups/wlan/iwlwifi/BoardConfig.mk
+++ b/groups/wlan/iwlwifi/BoardConfig.mk
@@ -7,7 +7,10 @@ WPA_SUPPLICANT_VERSION := VER_2_1_DEVEL
 BOARD_WLAN_DEVICE := iwlwifi
 {{/libwifi-hal}}
 
-BOARD_WPA_SUPPLICANT_PRIVATE_LIB ?= lib_driver_cmd_intc
+#BOARD_WPA_SUPPLICANT_PRIVATE_LIB ?= lib_driver_cmd_intc
+
+BOARD_HOSTAPD_PRIVATE_LIB   := //hardware/intel/wlan/libwifihal/open/wpa_supplicant_8_lib:lib_driver_cmd_intc
+BOARD_WPA_SUPPLICANT_PRIVATE_LIB := //hardware/intel/wlan/libwifihal/open/wpa_supplicant_8_lib:lib_driver_cmd_intc
 
 BOARD_SEPOLICY_DIRS += $(INTEL_PATH_SEPOLICY)/wlan/load_iwl_modules
 BOARD_SEPOLICY_DIRS += $(INTEL_PATH_SEPOLICY)/wlan/iwlwifi
-- 
2.34.1

